frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/health-card.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { DashboardCard } from '@console/internal/components/dashboard/dashboard-card/card';
+import { DashboardCardBody } from '@console/internal/components/dashboard/dashboard-card/card-body';
+import { DashboardCardHeader } from '@console/internal/components/dashboard/dashboard-card/card-header';
+import { DashboardCardTitle } from '@console/internal/components/dashboard/dashboard-card/card-title';
+import { HealthBody } from '@console/internal/components/dashboard/health-card/health-body';
+import { HealthItem } from '@console/internal/components/dashboard/health-card/health-item';
+import { HealthState } from '@console/internal/components/dashboard/health-card/states';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { STORAGE_HEALTH_QUERIES } from '../../../constants/storage';
+import { CEPH_HEALTHY, CEPH_DEGRADED, CEPH_ERROR, CEPH_UNKNOWN } from '../../../constants/index';
+
+const CephHealthStatus = {
+  0: {
+    message: CEPH_HEALTHY,
+    state: HealthState.OK,
+  },
+  1: {
+    message: CEPH_DEGRADED,
+    state: HealthState.WARNING,
+  },
+  2: {
+    message: CEPH_ERROR,
+    state: HealthState.ERROR,
+  },
+  3: {
+    message: CEPH_UNKNOWN,
+    state: HealthState.ERROR,
+  },
+};
+
+const getCephHealthState = (ocsResponse): CephHealth => {
+  if (!ocsResponse) {
+    return { state: HealthState.LOADING };
+  }
+  const value = _.get(ocsResponse, 'data.result[0].value[1]');
+  return CephHealthStatus[value] || CephHealthStatus[3];
+};
+
+const HealthCard: React.FC<DashboardItemProps> = ({
+  watchPrometheus,
+  stopWatchPrometheusQuery,
+  prometheusResults,
+}) => {
+  React.useEffect(() => {
+    watchPrometheus(STORAGE_HEALTH_QUERIES.CEPH_STATUS_QUERY);
+    return () => {
+      stopWatchPrometheusQuery(STORAGE_HEALTH_QUERIES.CEPH_STATUS_QUERY);
+    };
+  }, [watchPrometheus, stopWatchPrometheusQuery]);
+
+  const queryResult = prometheusResults.getIn([STORAGE_HEALTH_QUERIES.CEPH_STATUS_QUERY, 'result']);
+
+  const cephHealthState = getCephHealthState(queryResult);
+
+  return (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>Health</DashboardCardTitle>
+      </DashboardCardHeader>
+      <DashboardCardBody isLoading={cephHealthState.state === HealthState.LOADING}>
+        <HealthBody>
+          <HealthItem state={cephHealthState.state} message={cephHealthState.message} />
+        </HealthBody>
+      </DashboardCardBody>
+    </DashboardCard>
+  );
+};
+
+export const HealthCardWithResources = withDashboardResources(HealthCard);
```

Comment:
As discussed in https://github.com/openshift/console/pull/1873#issuecomment-507736616, code under `packages` should converge towards "single default component export per file" convention.

If this was a default export, you wouldn't have to give it an explicit name, i.e.

```ts
export default withDashboardResources(HealthCard);
```

It's up to you.

You can always add `packages/ceph-storage-plugin/.eslintrc` that extends `packages/.eslintrc.js` and tweak rules as necessary, although in the long term, all code under `packages` should converge to the same conventions and rule config.